src/custom-elements/src/components/expansion-panel/expansion-panel.tsx

```diff
@@ -0,0 +1,91 @@
+import { Component, Prop, Watch } from '@stencil/core';
+
+const EP_MODES = [
+  'done',
+  'active',
+  'pending',
+  'disabled'
+];
+
+@Component({
+  tag: 'chi-expansion-panel',
+  styleUrl: 'expansion-panel.scss',
+  scoped: true
+})
+export class ExpansionPanel {
+  /**
+   * to set expansion panel state. Possible values are: {'done', 'active', 'pending' (default value), and 'disabled'}
+   */
+  @Prop({ reflectToAttr: true }) state = 'pending';
+
+  /**
+   * to set a step number next to the title of the panel
+   */
+  @Prop({ reflectToAttr: true }) step: string;
+
+  /**
+   * to set the title of the panel
+   */
+  @Prop({ reflectToAttr: true }) headline: string;
+
+  /**
+   * is the panel border-styled?
+   */
+  @Prop({ reflectToAttr: true }) bordered: boolean;
+
+  @Watch('state')
+  stateValidation(newValue: string) {
+    if (newValue && !EP_MODES.includes(newValue)) {
+      throw new Error(
+        `${newValue} is not a valid state for this component. If provided, valid values are 'done', 'active', 'pending' or 'disabled'; if not, 'pending' value will be set. `
+      );
+    }
+  }
+
+  componentWillLoad() {
+    this.stateValidation(this.state);
+  }
+
+  render() {
+    return (
+        <div
```

Comment:
@SergioQCL it looks that code is wrong indented. Can you take a look?

Reply:
@SergioQCL Use prettier for this.

Reply:
I indeed used prettier, and the result I got was that foxhole, but I've double checked it, so it should be ok now.